build/source/engine/vegNrgFlux.f90

```diff
@@ -538,6 +492,11 @@ subroutine vegNrgFlux(&
  ! ---------------------------------------------------------------------------------------
  err=0; message="vegNrgFlux/" ! initialize error control
 
+ ! Validate method for calculation of derivatives
+ if (ix_fDerivMeth /= analytical) then
+  err=20; message=trim(message)//'only analytical derivatives are supported -- numerical derivatives are no longer used'; return
+ end if
```

Comment:
So far I think I've seen this error as a single line, 4 lines and now 3 lines. Is there a reason to choose any particular one? Might be neater to make this consistent across all files